Three-D-Viewer-Server/src/main/java/com/main/server/exception/JwtAuthenticationException.java

```diff
@@ -0,0 +1,14 @@
+package com.main.server.exception;
+
+import org.springframework.security.core.AuthenticationException;
+
+public class JwtAuthenticationException extends AuthenticationException {
```

Comment:
please add ResponseStatus to exception and then in the ControllerAdvice add the new exception handler which will bind the response status by this annotation.